Configuration/AlCa/python/autoAlca.py

```diff
@@ -7,7 +7,8 @@
                   'DoubleMu'       : 'MuAlCalIsolatedMu+MuAlOverlaps+DtCalib+TkAlZMuMu',
                   'DoubleMuon'     : 'TkAlZMuMu+MuAlCalIsolatedMu+MuAlOverlaps+MuAlZMuMu+DtCalib',
                   'DoubleMuParked' : 'MuAlCalIsolatedMu+MuAlOverlaps+DtCalib+TkAlZMuMu',
-                  'HLTPhysics'     : 'SiStripCalMinBias+TkAlMinBias',
+                  'HLTPhysics'     : 'SiStripCalMinBias+TkAlMinBias+HcalCalIsoTrkFilter',
+                  'Cosmics'        : 'TkAlCosmics0T',
```

Comment:
@bsunanda the `Cosmics` key appears here twice. Please remove the unnecessary one.